acceptance/bundle/python/resource-ordering/resources.py

```diff
@@ -0,0 +1,23 @@
+from databricks.bundles.core import Bundle, Resources
+
+# databricks.yml defines the order of resource loading
+
+
+def load_resources_2() -> Resources:
+    resources = Resources()
+    resources.add_job(
+        resource_name="my_job_2",
+        job={"name": "Job 2"},
+    )
+
+    return resources
+
+
+def load_resources_1() -> Resources:
```

Comment:
I don't see how the ordering comment for these functions is significant.

Reply:
Agree. There are minor differences in output, but overall we are testing resource loading, so I've renamed the test and removed this comment